src/Audio/SoundEffectInstance.cs

```diff
@@ -263,6 +263,12 @@ public void Apply3D(AudioListener listener, AudioEmitter emitter)
 
 		public void Apply3D(AudioListener[] listeners, AudioEmitter emitter)
 		{
+            if (listeners.Length == 0)
```

Comment:
I just took a second look and this actually isn't valid - if listeners is empty you get ArgumentOutOfRange. [1] is valid, however, so you can just take this first check out entirely.